src/translations/en.json

```diff
@@ -8,5 +8,57 @@
     "log_out": "Log out",
     "mailbox": "Mailbox",
     "shopping_list": "Shopping list"
+  },
+  "state": {
+    "on": "On",
+    "off": "Off",
+    "home": "Home",
+    "not_home": "Away",
+    "unknown": "Unknown",
+    "open": "Open",
+    "opening": "Opening",
+    "closed": "Closed",
+    "closing": "Closing",
+    "playing": "Playing",
+    "paused": "Paused",
+    "idle": "Idle",
+    "standby": "Standby",
+    "armed": "Armed",
+    "disarmed": "Disarmed",
+    "armed_home": "Armed Home",
+    "armed_away": "Armed Away",
+    "armed_night": "Armed Night",
+    "pending": "Pending",
+    "arming": "Arming",
+    "disarming": "Disarming",
+    "triggered": "Triggered",
+    "locked": "Locked",
+    "unlocked": "Unlocked",
+    "unavailable": "Unavailable",
+    "ok": "OK",
+    "problem": "Problem",
+    "dry": "Dry",
+    "wet": "Wet",
+    "clear": "Clear",
+    "detected": "Detected",
+    "closed": "Closed",
+    "open": "Open",
+    "safe": "Safe",
+    "unsafe": "Unsafe"
+  },
+  "state_badge": {
```

Comment:
Let's also put these under a domain. `state_badge.device_tracker.home` for example

Reply:
I think the binary sensor entries would be safe to separate out, since they only exist for display on the frontend.

However, I'm more concerned about the other domain states. We definitely have some states that are shared between domains, (i.e. STATE_OFF, STATE_ON), but more importantly, some of the domain states have been co-opted in a few odd places by other platforms. Some examples:

https://github.com/home-assistant/home-assistant/blob/2a28f7c4b506968255fea5a96bfe2c8f2ffd271d/homeassistant/components/media_player/roku.py#L138
https://github.com/home-assistant/home-assistant/blob/2a28f7c4b506968255fea5a96bfe2c8f2ffd271d/homeassistant/components/sensor/modem_callerid.py#L118
https://github.com/home-assistant/home-assistant/blob/2a28f7c4b506968255fea5a96bfe2c8f2ffd271d/homeassistant/components/sensor/mqtt_room.py#L74

I think that if we want to break these states out here, we should first do a backend refactoring to move these state constants into the domain components. Thoughts?

Reply:
I think it is OK for backend to share states.
The frontend should still have separate keys. Lokalise supports linking keys, so there is no need to translate several times unless there is real need.

Reply:
OK, I think I'm on board with that. So in our schema we'll have some duplicates across domains, but different languages might need different translations for different domain constants. 👍